framework/src/main/java/org/checkerframework/framework/util/CheckerMain.java

```diff
@@ -58,9 +66,15 @@ public static void main(String[] args) {
         System.exit(exitStatus);
     }
 
-    /** The path to the annotated jdk jar to use. */
+    /** In case of Java 8, the path to the annotated jdk.jar. Otherwise {@code null}. */
     protected final File jdkJar;
```

Comment:
To improve the documentation, you could add `@Nullable` here and on the new field type.

Reply:
Done.